libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/EntityAssignment.cs

```diff
@@ -61,6 +61,15 @@ public class EntityAssignment
         [JsonProperty("raisedCount")]
         public uint RaisedCount { get; set; } = 0;
 
+        /// <summary>
+        /// Gets or sets the expected properties when assignment was made.
+        /// </summary>
+        /// <value>
+        /// Expected properties.
+        /// </value>
+        [JsonProperty("expectedProperties")]
+        public IReadOnlyCollection<string> ExpectedProperties { get; set; }
```

Comment:
>IReadOnlyCollection [](start = 15, length = 19)

Ummm..how does this work with json serializer?

Reply:
Our style rules complain about List and this seems to work fine.